servicetalk-opentracing-inmemory/build.gradle

```diff
@@ -18,8 +18,10 @@ apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
 dependencies {
   api project(":servicetalk-opentracing-inmemory-api")
+  api project(":servicetalk-concurrent")
```

Comment:
we have `BiIntPredicate` and `Function3` (and friends) in `concurrent.api` ... any reason why `TriConsumer` isn't in the same place?